docs/mathematics/Kth-smallest.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: K-th smallest element in an array
+slug: /mathematics/Kth-smallest
+---
+
+## Introduction
+
+In this article we will be given a array and a number K we have to find the Kth smallest element in an array. 
+
+## Explanation 
+In this first we have to sort the array in ascending order. Then print Kth index(if 1 based indexing that means array index starts from 1) or (K-1)th index(if 0 based indexing that means array index starts from 0).
```

Comment:
Use proper formatting for numbers and all

Reply:
done